packages/markdown/html.lua

```diff
@@ -0,0 +1,44 @@
+local base = require("packages.base")
+local utils = require("packages.markdown.utils")
+
+local package = pl.class(base)
+package._name = "markdown.html"
+
+local passthroughs = {
+  "sup",
```

Comment:
SILE does not have these, right? We'd need to map them
- to raise/lower (in the general case)
- or perhaps even, initially to the package-provided textsuperscript/textsubscript (if containing only text).

Reply:
It has them.

Reply:
It doesn't seem so (at least, there's no "sup"/"sub" in the base and plain class), they only exist it seems in math mode.

Reply:
Heh, with all due respect, you do so much work it seems you forget some of it!

https://github.com/Omikhleia/markdown.sile/blob/cd90084ab77f5ed82764465354443d1f3018cd7c/packages/markdown/commands.lua#L97

Reply:
> Heh, with all due respect, you do so much work it seems you forget some of it!

Heh :) But in that case the commands are `textsuperscript` and `textsubscript` then (not `sup` and `sub`, hence my initial question).

In the current implementation, they'd work if the content is text-only. (If it's not, they ought to fail, although we may reconsider our options)

Reply:
Huh, strange, my document still compiles though. But you're right, the text is not superscripted.